exercises/rotational-cipher/cannonical-data.json

```diff
@@ -0,0 +1,72 @@
+{
+    "#": [
+        "The tests are a series of rotation tests and a bonus test: ",
+        "* Rotational tests",
+        "* non-ASCII downgrade tests (bonus)"
+    ],
+    "rotate": {
+        "description": ["Test rotation from English to ROTn"],
+        "cases": [
+            {
+                "description": "rotate a by 1",
+                "text": "a",
+                "shift_key": 1,
+                "expected": "b"
+            },
+            {
+                "description": "rotate a by 26, same output as input",
+                "text": "a",
+                "shift_key": 26,
+                "expected": "a"
+            },
+            {
+                "description": "rotate a by 0 same input as output",
```

Comment:
seems weird to have the wording be different than above. how about making them consistent?